package.json

```diff
@@ -1306,11 +1313,18 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "webpack --env.production --config-name webviewIndex.js && webpack --env.production --config-name activityBar-webviewIndex.js && webpack --env.production --config-name extension && npx webpack-cli --config browser.webpack.config --env.production --config-name extension --mode none",
+    "webpack-description": "webpack --config-name webviewIndex.js",
+    "webpack-active-pr": "webpack --config-name activityBar-webviewIndex.js",
+    "webpack-create": "webpack --config-name createPR-webviewIndex.js",
+
+    "webpack:dev": "yarn run webpack-description --env.development && yarn run webpack-active-pr --env.development && yarn run webpack-create --env.production",
```

Comment:
Copy paste error? At the end of the line, should it be `--env.development`?

Reply:
And thanks for breaking this up, it's way more readable.

Reply:
Thanks for catching that!

Reply:
I realize I missed updating the `compile-web` command below, do you know why we sometimes use `webpack-cli` and sometimes use `webpack`? I'm also not sure why we set mode to `none` there and for the extension in prepublish, from looking at the documentation, the major difference between modes is if names get mangled or not. Maybe related to the problem with node-fetch?